.gitignore

```diff
@@ -22,3 +22,4 @@ source/.jekyll-metadata
 !.vscode/tasks.json
 *.suo
 tags*
+Gemfile.lock
```

Comment:
Please only include files that are relevant to this PR. Don't push gitignore or Gemfile.lock.